Module-6-Assignment/AccountContactsTest.cls

```diff
@@ -0,0 +1,59 @@
+/**
+ * Test class to test the functionality of AccountContacts Class 
+ */
+@isTest
+public class AccountContactsTest 
+{
+	@isTest
+    private static void testWithOneContactId()
```

Comment:
Add a description of what the test is doing

Reply:
Done